frontend/beCompliant/src/pages/QuestionPage.tsx

```diff
@@ -107,7 +106,7 @@ export const QuestionPage = () => {
             isAnswerEdited={isAnswerEdited}
             setIsAnswerEdited={setIsAnswerEdited}
           />
-          <QuestionInfoBox question={question} tableId={tableId} />
+          <QuestionInfoBox question={question} tableId={tableId ?? ''} />
```

Comment:
Er det bedre å returnere/kaste en feilmelding høyere opp istedenfor å sende med en tom string her?
```suggestion
          <QuestionInfoBox question={question} tableId={tableId} />
```